crates/mockiato-codegen/src/syn_ext.rs

```diff
@@ -9,3 +9,28 @@ impl PathExt for Path {
         Some(&self.segments.iter().nth(0)?.ident)
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use syn::punctuated::Punctuated;
+    use syn::{parse_quote, Ident, Path};
+
+    #[test]
+    fn returns_none_if_path_is_empty() {
+        let path = Path {
+            leading_colon: None,
+            segments: Punctuated::new(),
+        };
+
+        assert!(path.first_segment_as_ident().is_none());
+    }
+
+    #[test]
+    fn returns_first_segment_without_params() {
+        let path: Path = parse_quote!(Foo<'a>::Bar::Baz);
+        let expected_ident: Ident = parse_quote!(Foo);
+
+        assert_eq!(&expected_ident, path.first_segment_as_ident().unwrap(),);
```

Comment:
What is that trailing comma doing there? Shouldn't rustfmt remove it?

Reply:
No rustfmt doesn't remove trailing commas here 😞. Addressed in 8645106